filecoin-proofs/src/param.rs

```diff
@@ -25,6 +25,13 @@ pub struct ParameterData {
     pub sector_size: u64,
 }
 
+// Deserializes bytes from the provided path into a ParameterMap
+pub fn read_parameter_map_from_str(string: &str) -> Result<ParameterMap> {
```

Comment:
If this function is not used outside of `paramfetch.rs`, I see no need to put it in `param.rs`. Do you?